misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/XMLTemplateEvaluator.java

```diff
@@ -54,10 +55,9 @@ public BExpressionValue evaluate() throws EvaluationException {
             for (Node memberNode : syntaxNode.content()) {
                 xmlStrBuilder.append(memberNode.toSourceCode());
             }
-            Value xmlStrValue = VMUtils.make(context, xmlStrBuilder.toString()).getJdiValue();
-
-            RuntimeStaticMethod fromStringMethod = getRuntimeMethod(context, FROM_STRING_CLASS, XML_FROM_STRING_METHOD,
-                    Collections.singletonList(B_STRING_CLASS));
+            Value xmlStrValue = EvaluationUtils.getAsJString(context, xmlStrBuilder.toString());
+            RuntimeStaticMethod fromStringMethod = getRuntimeMethod(context, B_TYPE_CONVERTER_CLASS,
```

Comment:
Shouldn't this variable name be `strToXmlMethod` after the changes?

Reply:
Fixed in ae66fd25bdd34c10affa694460255a0f3bf4983f